keanu-python/keanu/base.py

```diff
@@ -19,7 +19,7 @@ def __getattr__(self, k):
             raise AttributeError("{} has no attribute {}".format(self.__class__, k))
 
         java_name = _to_camel_case_name(k)
-        warnings.warn("\"{}\" is not implemented so Java API \"{}\" was called directly instead".format(k, java_name))
+        logging.warning("\"{}\" is not implemented so Java API \"{}\" was called directly instead".format(k, java_name))
```

Comment:
I added this line after discussing with Joanne - `warnings.warn` raises a warning that displays in pytest output, whereas `logging.warning` just logs to stdout (which is only visible when pytest fails a test)